scylla/tests/ccm-integration/node_config.rs

```diff
@@ -0,0 +1,365 @@
+use core::str;
+use serde_yaml::Value;
+use std::collections::HashMap;
+
+/// Represents data stored in scylla.yaml or cassandra.yaml
+#[derive(Debug, Clone)]
+pub(crate) enum NodeConfig {
+    Null,
+    Bool(bool),
+    Int(i64),
+    Float(f64),
+    String(String),
+    List(Vec<NodeConfig>),
+    Map(HashMap<String, NodeConfig>),
+}
+
+impl Default for NodeConfig {
+    fn default() -> NodeConfig {
+        let mut val = HashMap::new();
+        val.insert(
+            "endpoint_snitch".to_string(),
+            NodeConfig::String(
+                "org.apache.cassandra.locator.GossipingPropertyFileSnitch".to_owned(),
+            ),
+        );
+        NodeConfig::Map(val)
+    }
+}
```

Comment:
Its not obvious to me why this specific value is the only one that must be in the default config. Please justify it in the comment.

Reply:
Btw is there a reason for making this a dynamic type instead of a struct with fields corresponding to config values, besides the fact that it would probably be more work?